packages/jssdk/shuffle/zkShuffle.ts

```diff
@@ -1,3 +1,4 @@
+import { FileType } from './zkShuffle';
```

Comment:
Why should we import `FileType` if we define it at Line 26?

Reply:
Sorry, Vscode plugin automatically import it , i have deleted it